### ISSUE TITLE
stuck at changing status

### ISSUE THREAD

**Issue by nightfreak** [codetwice/homebridge-http-securitysystem#3]
hey there,

when opening the home app, the status is shown correctly.
But when I change the status, my security system gets the command correctly and then the homekit app stays showing "activating" and never gets the new status.

is there a fix for this?

**Comment by codetwice** [comment]
Unfortunately I do not have my HomeKit setup running any more so I can't test the behaviour on the new Home app released as part of iOS10. 
Security devices in HomeKit have 2 different state properties: a target state and a current state. When you are turning on your alarm, HomeKit sets the target state, which then, later on, causes a change in the current state as well. 
The app I used earlier (MyTouchHome) also had this defect. It issued the target state change when I turned on my alarm, it also updated the target state display immediately but did not bother reloading the current state later on. 

Unfortunately I do not have access to a security device so I cannot properly debug the Home app's behaviour and can't fix the problem at the moment. if there is anyone out there willing to do this, I would be happy to invite him to the project.
